Wrapping/Generators/Python/CMakeLists.txt

```diff
@@ -293,211 +293,6 @@ macro(itk_setup_swig_python type base_name interface_file python_file cpp_file d
 endmacro()
 
 
-macro(itk_end_wrap_module_python)
```

Comment:
This commit seems to move code from one place to another, but commit message implied only removal.

Reply:
Fixed by squashing and changing several commit messages.